ui/app/components/pending-tx-details.js

```diff
@@ -29,8 +29,10 @@ PTXP.render = function () {
   var account = props.accounts[address]
   var balance = account ? account.balance : '0x0'
 
+  var gasMultiplier = txParams.gasMultiplier
   var gasCost = new BN(ethUtil.stripHexPrefix(txParams.gas || txData.estimatedGas), 16)
   var gasPrice = new BN(ethUtil.stripHexPrefix(txParams.gasPrice || '0x4a817c800'), 16)
+  gasPrice = new BN(parseFloat(gasPrice.toString()) * gasMultiplier)
```

Comment:
I guess this would be another place to try to use the same logic (use BN for the math, don't string to num to BN, that loses precision).